documentation/book/proc-upgrading-brokers-newer-kafka.adoc

```diff
@@ -2,41 +2,51 @@
 //
 // assembly-upgrading-kafka-versions.adoc
 
-[id='proc-upgrading-brokers-newer-kafka-0-9-0-to-0-10-0-{context}']
+[id='proc-upgrading-brokers-newer-kafka-{context}']
 
-= Upgrading brokers to a newer Kafka version
+= Upgrading Kafka brokers and client applications
```

Comment:
I feel the more descriptive title was helpful. Please consider changing it to:

Upgrading Kafka brokers and client applications to a new Kafka version

Reply:
Have changed, but I get uncomfortable with long headings. :-)